src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -41,6 +41,14 @@ configurationRegistry.registerConfiguration({
 		'terminal.integrated.fontFamily': {
 			'description': nls.localize('terminal.integrated.fontFamily', "The font family used by the terminal (CSS font-family format), this defaults to editor.fontFamily's value."),
 			'type': 'string'
+		},
+		'terminal.integrated.fontSize': {
```

Comment:
Font size should have a default here, otherwise auto-complete does this.

![image](https://cloud.githubusercontent.com/assets/2193314/15760084/816ab9b0-28c6-11e6-8262-e26f7e994a58.png)